ui/src/scss/base.scss

```diff
@@ -31,3 +31,15 @@ body {
   @extend %icon;
   @include maas-icon-edit($color-mid-dark);
 }
+
+.u-text--light {
```

Comment:
This class isn't used

Reply:
It's used here: https://github.com/canonical-web-and-design/maas-ui/pull/69/files#diff-1ee9d1795a53072cc392e1124036416bR89.